x/gamm/twap/keeper.go

```diff
@@ -23,5 +23,5 @@ func NewKeeper(storeKey sdk.StoreKey, transientKey *sdk.TransientStoreKey, param
 
 // TODO: make this read from a parameter, or hardcode it.
 func (k *Keeper) PruneEpochIdentifier(ctx sdk.Context) string {
-	return "daily"
+	return "day"
```

Comment:
we should make these constants :)

Reply:
Also made this to address the TODO: https://github.com/osmosis-labs/osmosis/issues/2289